test/helper.rb

```diff
@@ -158,6 +158,14 @@ def md.directory?
       'C:/etc/postgresql/9.5/main/pg_ident.conf' => mockfile.call('pg_ident.conf'),
       '/etc/postgresql/9.5/main' => mockfile.call('9.5.main'),
       '/var/lib/postgresql/9.5/main' => mockfile.call('var.9.5.main'),
+      '/etc/hosts.allow' => mockfile.call('hosts.allow'),
```

Comment:
There are a bunch of changes in `test/helper.rb` that have nothing to do with this change. I think you need to rebase onto master. Feel free to squash your commits at this point as well since we're pretty much at final review.